enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1220 @@
+We could implement change control without `disabledUntil`, `pausedUntil`, `exclude`, and perhaps more. However,
+it is risky to impose a single opinionated workflow onto the wide variety of consumers of the platform. The workflows
+described in this enhancement are not intended to be exotic or contrived but situations in which flexibility
+in our configuration can achieve real world, reasonable goals. 
+
+`disabledUntil` is designed to support our Service Delivery team who, on occasion, will need
+to be able to bypass configured change controls. The feature is easy to use, does not require 
+deleting or restoring customer configuration (which may be error-prone), and can be safely 
+"forgotten" after being set to a date in the future.
+
+`pausedUntil`, among other interesting possibilities, offers a cluster lifecycle administrator the ability
+to stop a problematic update from unfolding further. You may have watched a 100 node
+cluster roll out a bad configuration change without knowing exactly how to stop the damage
+without causing further disruption. This is not a moment when you want to be figuring out how to format
+a date string, calculating timezones, or copying around cluster configuration so that you can restore
+it after you stop the bleeding.
+
+### Implement change control, but do not implement the Manual and/or Assisted strategy for MachineConfigPool
+Major enterprise users of our software do not update on a predictable, recurring window of time. Updates
+require laborious certification processes and testing. Maintenance schedules will not serve these customers
+well. However, these customers may still benefit significantly from the change management concept --
+unexpected / disruptive worker node drains and reboots have bitten even experienced OpenShift operators
+(e.g. a new MachineConfig being contributed via gitops).
+
+These strategies inform decision-making through metrics and provide facilities for fine-grained control 
+over exactly when material change is rolled out to a cluster. 
+
+The Assisted strategy is also specifically designed to provide a foundation for 
+the forthcoming `oc adm update worker-nodes` verbs. After separating the control-plane and
+worker-node update phases, these verbs are intended to provide cluster lifecycle administrators the 
+ability to easily start, pause, cancel, and even rollback worker-node changes.
+
+Making accommodations for these strategies should be a subset of the overall implementation
+of the MaintenanceSchedule strategy and they will enable a foundation for a range of 
+different persons not served by MaintenanceSchedule.
+
+### Use CRON instead of RRULE
+The CRON specification is typically used to describe when something should start and 
+does not imply when things should end. CRON also cannot, in a standard way,
+express common semantics like "The first Saturday of every month."

```

Comment:
How would you do this with RRULE?

Reply:
`FREQ=Monthly;BYDAY=Sa;BYSETPOS=-1,1;`

Reply:
May be worth including as an example in the body

Reply:
Adding in next push in a new section that introduces terminology early.